helper_crates/vtable/src/vrc.rs

```diff
@@ -291,6 +291,12 @@ impl<VTable: VTableMetaDropInPlace + 'static, X> Default for VWeak<VTable, X> {
     }
 }
 
+impl<VTable: VTableMetaDropInPlace + 'static, X> PartialEq for VWeak<VTable, X> {
```

Comment:
If you add something that you use, please increase the version we we don't forget to publish vtable.
But Should we really have that, there is already VWeak::ptr_eq
Vweak doesn't have PArtialEq for the same reason the std's Weak doesn't have it. Which I think it that it could be confused with a comparison of the content instead of just the pointer.